src/uwtools/tests/fixtures/jedi.yaml

```diff
@@ -0,0 +1,19 @@
+forecast length: P2D
```

Comment:
This file isn't used for tests as far as I can tell. Perhaps we can remove it.

Reply:
I was thinking the same thing. Perhaps move it to be included under what already exists for the shared/docs/jedi.yaml file

Reply:
As we use more inline `pytest` fixtures, we should probably periodically do a sweep through the external fixture files and see if there are any that are no longer used by any unit tests. I've done it once or twice in the past and will try to do another check someday soon.